reconcile/utils/aws_api.py

```diff
@@ -874,6 +883,32 @@ def get_vpcs_details(self, account, tags=None, route_tables=False):
 
         return results
 
+    def get_amis_details(self, account, owner_account, regex):
```

Comment:
Can we have type hints here, please?

Reply:
i'll add type hints, also wrote it in the description